src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Unix.cs

```diff
@@ -63,11 +61,16 @@ private void AcceptCompletionCallback(IntPtr acceptedFileDescriptor, byte[] sock
 
         internal unsafe SocketError DoOperationAccept(Socket socket, SafeCloseSocket handle, SafeCloseSocket acceptHandle, out int bytesTransferred)
         {
+            if (_buffer != null)
+            {
+                throw new PlatformNotSupportedException(SR.net_sockets_accept_receive_notsupported);
+            }
+
```

Comment:
Is there a functional difference between accepting with a buffer vs accepting and then separately receiving?  I'm wondering if we should just do the work now to do the receive, or if that's actually not equivalent behavior.